modules/op-release-notes-1-19.adoc

```diff
@@ -0,0 +1,396 @@
+// This module is included in the following assemblies:
+// * release_notes/op-release-notes-1-19.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="op-release-notes-1-19_{context}"]
+= Release notes for {pipelines-title} General Availability 1.19
+
+With this update, {pipelines-title} General Availability (GA) 1.19 is available on {OCP} 4.15 and later versions.
+
+[id="new-features-1-19_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.19:
+
+[id="pipelines-new-features-1-19_{context}"]
+=== Pipelines
+
+* With this update, you can now specify custom `securityContext` settings under the `EventListener` YAML specification. When custom `securityContext` is enabled, user-defined values override the default configuration. If no custom values are provided, the default `securityContext` setting is applied automatically.
++
+.Example configuration for enabling custom `securityContext` in the `EventListener` YAML spec
+[source,yaml]
+----
+spec:
+  serviceAccountName: tekton-triggers-example-sa
+  resources:
+    kubernetesResource:
+      spec:
+        template:
+          spec:
+            securityContext:
+              runAsNonRoot: true
+            containers:
+            - resources:
+                requests:
+                  memory: "64Mi"
+                  cpu: "250m"
+                limits:
+                  memory: "128Mi"
+                  cpu: "500m"
+              securityContext:
+                readOnlyRootFilesystem: true
+----
+
+* With this update, you can configure custom database credentials for {tekton-results} using the `TektonConfig` custom resource (CR). This eliminates the need to rely on default PostgreSQL secrets that use default usernames and passwords.
+
+* With this update, {pac} no longer creates a pending status on GitHub Pull Requests when an unauthorized bot user attempts to trigger a `PipelineRun`. Instead of generating a blocking status check, such requests are now silently disallowed.
+
+* With this update, a new metric, `pipelines_as_code_git_provider_api_request_count`, tracks the number of API calls made by {pac} to Git providers, such as GitHub, GitLab, and Gitea. The metric also helps to monitor API rate limit usage across Git providers, namespaces, event types, and repositories.
+
+* With this update, repository URLs are now validated during creation to ensure they are properly formatted and use valid schemes, such as `http` or `https`. This enhancement helps prevent misconfigurations and runtime errors.
```

Comment:
```suggestion
* With this update, Repository URLs are now validated during creation to ensure they are properly formatted and use valid schemes, such as `http` or `https`. This enhancement helps prevent misconfigurations and runtime errors.
```
Capitalizing `Repository` to disambiguate between a code repository and a Pipelines as Code Repository Custom Resource

Reply:
It looks like k8s resources are generally not capitalized in the rest of the document. If there is another way to disambiguate this that would be useful. There's no other context in this bullet to indicate this is related to Pipelines as Code, and in my experience confusing and conflating a git repository and a PaC Repository CR is very common when talking with users.